database_benchmarking_suite/src/main/scala/uk/ac/ebi/eva/benchmarking_suite/mongodb/MongoDBWriteSampler.scala

```diff
@@ -35,13 +36,13 @@ class MongoDBWriteSampler() extends AbstractSampler {
         Duration.Inf)
     }
 
-    if (counter != numInsertsPerThread) {
-      val timestamp = System.currentTimeMillis()
-      val accession_id = "acc_%d_%d_%s".format(threadNum, counter, timestamp)
+    if (counter <= numInsertsPerThread) {
```

Comment:
The condition now accepts equals but it didn't before, is this intentional?

Reply:
sorry. my bad! i have updated the code!!